### PR TITLE
Add TS.REVRANGE, TS.MREVRANGE

### PR DESCRIPTION
**Pull request by felipecsl**
If this looks good for folks, I can go ahead and add unit tests if needed before merging

### CODE REVIEW

**Comment by ashtul** [comment]
Hi @felipecsl ,
Code looks good.
Do you plan on adding the unit tests?

**Comment by felipecsl** [comment]
@ashtul yup! I can do that in the next few days!

File: pom.xml [hunk 1]
```diff
@@ -89,8 +89,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.1</version>
 				<configuration>
-					<source>1.7</source>
-					<target>1.7</target>
+					<source>8</source>
+					<target>8</target>
```
[the remark this answers is not in the document]
Comment (filipecosta90) [reply pom.xml]: Agree that we might not need to bump version. Reverted :)

**Comment by filipecosta90** [comment]
LGTM @felipecsl. Even tough we're missing the MREVRANGE tests those will be added by #26 .
Merging :) Thank you !

**Review by filipecosta90 — APPROVED** [review]

**Pull request merged**